joomla/joomla-cms
# [4.0] Header module dropdown styling

Pull Request for Issue https://github.com/joomla/joomla-cms/issues/27541.

### Summary of Changes

Tweaks styling in some dropdowns.

### Testing Instructions

> 
>     Go to the #__postinstall_messages table
>     Replace any of the language constants in the title_key column with a faily long string.
>     Go to the Joomla backend and open the "Post Installation Messages" dropdown
> 
> You'll notice there's no right padding on the items.

### Expected result

![untitlesd](https://user-images.githubusercontent.com/7325021/72794938-787ba300-3c45-11ea-83db-097e1bc5ec17.png)


### Actual result

![](https://user-images.githubusercontent.com/2019801/72430953-85068400-378b-11ea-9b10-2010dffed511.png)

### Documentation Changes Required
